Readme.md

```diff
@@ -1,3 +1,7 @@
+<a href="docs/contributing.md" alt="Hacktoberfest"><img src="https://badgen.net/badge/hacktoberfest/friendly/pink" /></a>
```

Comment:
The link for this badge doesn’t work — probably because they don’t have a contributing.md file in this repository.

Reply:
I corrected it. Please have a look.

Reply:
For some reason the "Open Source" badge seems to have a transparent background and doesn't really show up. I thought it had a colored background before, so it's strange that now it doesn't.

Reply:
Sorry, I couldn't understand the error. Can you suggest the solution so that I can work on it.

Reply:
<img width="275" alt="hard-to-read-text" src="https://user-images.githubusercontent.com/3969255/95533298-ab162900-0997-11eb-9eea-375d29639e15.PNG">

The 3rd badge is unreadable.

Reply:
![Screenshot from 2020-10-09 08-29-10](https://user-images.githubusercontent.com/50957556/95536062-c15ccd00-0a09-11eb-97cd-d462de04b14c.png)
 But its working on mine brother.

Reply:
Interesting. It’s working with my copy of chrome but not Firefox. So it’s either the svg does something strange, or a bug in firefox.

Reply:
Any solution??

Reply:
It works with Firefox version 53, stops working at least in firefox versions 54-68. The firefox nightly build (not sure about the age... I'm under the impression browserling.com hasn't added new versions in a while) that draws it correctly.